jsettlers.main.android/src/main/java/jsettlers/main/android/gameplay/gamemenu/GameMenuDialog.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2017
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+package jsettlers.main.android.gameplay.gamemenu;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.WindowManager;
+import android.widget.SeekBar;
+
+import jsettlers.main.android.R;
+import jsettlers.main.android.databinding.DialogGameMenuBinding;
+import jsettlers.main.android.mainmenu.MainActivity_;
+
+public class GameMenuDialog extends DialogFragment {
+
+    private GameMenuViewModel viewModel;
+
+    public static GameMenuDialog create() {
+        return new GameMenuDialog();
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        GameMenuViewModelFactory gameMenuViewModelFactory = new GameMenuViewModelFactory(getActivity().getApplication());
```

Comment:
better use requireApplication instead of getApplication, since getApplication might produce a NullPointer (should be shown as warning by Android Studio)